### PR TITLE
[feat] Renaming

### PR DESCRIPTION
**Pull request by mcdoker18**

### CODE REVIEW

File: source/cmp_legionctl.md [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
 
-```bash
-legionctl inspect --edi <edi-url> --token <your-token>
-```
-
-Another option save the token in config file. Further you can use legionctl commands without `edi` and
-`token` parameters. For example:
+You should open an API server URL in a browser to get the login command.
+The command already contains your token.
+Copy and past provided command into your shell.
```
Comment (maxkochubey) [inline source/cmp_legionctl.md]:
```suggestion
Copy and paste provided command into your shell. 
```

File: source/ref_model_format.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Model Format
 
-The **Legion Model Artifact Format** (LMAF) describes a format to package, store, and transport ML models.
+The **Odahu Model Artifact Format** (LMAF) describes a format to package, store, and transport ML models.
```
Comment (maxkochubey) [inline source/ref_model_format.md]: OMAF already?

File: source/comp_model_feedback.rst [hunk 3]
```diff
@@ -67,31 +67,31 @@ The first file contains meta-information about request and response:
...
+            ":authority": "edge.flow-test.epm.kharlamov.biz",
             ":path": "/api/model/invoke",
             ":method": "POST",
             "user-agent": "python-requests/2.22.0",
             "x-b3-traceid": "1bd630537a9a54c9d3997150ca540d01",
             "x-request-id": "85252432-8d26-9b5f-bed3-e9ceeafc688c",
             "x-b3-sampled": "1",
             "x-envoy-external-address": "1.1.1.1",
-            "x-forwarded-host": "edge.legion-test.epm.kharlamov.biz",
+            "x-forwarded-host": "edge.flow-test.epm.kharlamov.biz",
```
Comment (karbyshevds) [inline source/comp_model_feedback.rst]: maybe we should remove `epm.kharlamov.biz` mentions ?

File: source/dev_testing.md [hunk 4]
```diff
@@ -1,42 +1,41 @@
...
 * You can optionally override the following parameters in `.env` file:
   * `CLUSTER_NAME`
   * `HIERA_KEYS_DIR`
   * `SECRET_DIR`
   * `CLOUD_PROVIDER`
   * `EXPORT_HIERA_DOCKER_IMAGE`
-  * `LEGION_PROFILES_DIR`
-* Executes `make export-hiera` command in the legion dir.
+  * `ODAHUFLOW_PROFILES_DIR`
+* Executes `make export-hiera` command.
```
Comment (karbyshevds) [inline source/dev_testing.md:24]: do we need this information in public document?

File: source/ref_network_connectivity.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-* Connection between [EDI](./cmp_edi.md) and Docker Registry that is used for storing trained models - for inspecting meta-information about images before deploy
-* Connection between cluster's Ingress controller (nginx-ingress or etc.) and [EDGE](./cmp_edge.md) (for model API) and [EDI](./cmp_edi.md) (for manage API)
+* Connection between [Operator](./comp_operator.md) and model training and model deployment instances - for inspecting training and deployed models
+* Connection between [API](./comp_api.md) and model deployment instances - for inspecting deployed models
+* Connection between [EDGE](./comp_edge.md) and model deployment instances - for routing model API traffic
+* Connection between [API](./comp_api.md) and Docker Registry that is used for storing trained models - for inspecting meta-information about images before deploy
+* Connection between cluster's Ingress controller (nginx-ingress or etc.) and [EDGE](./comp_edge.md) (for model API) and [API](./comp_api.md) (for manage API)
 * Connection between training instances and statsd (in external namespace) - for storing train metrics
```
Comment (maxkochubey) [inline source/ref_network_connectivity.md]: Is the info about statsd actual yet?

File: source/gs_feedback_loop.rst [hunk 6]
```diff
@@ -83,31 +83,31 @@ First file contains meta information about request and response (name is like ``
...
+            ":authority": "edge.odahu-test.epm.kharlamov.biz",
             ":path": "/api/model/invoke",
             ":method": "POST",
             "user-agent": "python-requests/2.22.0",
             "x-b3-traceid": "1bd630537a9a54c9d3997150ca540d01",
             "x-request-id": "85252432-8d26-9b5f-bed3-e9ceeafc688c",
             "x-b3-sampled": "1",
             "x-envoy-external-address": "1.1.1.1",
-            "x-forwarded-host": "edge.legion-test.epm.kharlamov.biz",
+            "x-forwarded-host": "edge.odahu-test.epm.kharlamov.biz",
```
Comment (karbyshevds) [inline source/gs_feedback_loop.rst]: Maybe we should delete mentions of `epm.kharlamov.biz` ?

File: source/cmp_legionctl.md (hunk 1 above)
Comment (mcdoker18) [reply source/cmp_legionctl.md]: Fixed

File: source/ref_model_format.md (hunk 2 above)
Comment (mcdoker18) [reply source/ref_model_format.md]: Fixed

File: source/ref_network_connectivity.md (hunk 5 above)
Comment (mcdoker18) [reply source/ref_network_connectivity.md]: Removed

**Review by maxkochubey — APPROVED** [review]

**Review by karbyshevds — APPROVED** [review]

File: source/gen_distros.rst [hunk 7]
```diff
@@ -2,75 +2,74 @@
...
 
+.. csv-table::
+   :header: "Package name", "Repository", "Description"
+   :widths: 20, 30, 40
 
+   "odahu-flow-cli", "odahu/odahu-flow", "Odahu-flow CLI"
+   "odahu-flow-sdk", "odahu/odahu-flow", "Odahu-flow SDK"
+   "odahu-flow-jupyterlab-plugin", "odahu/odahu-flow-jupyterlab-plugin", "Jupyterlab with the Odahu-flow plugin"
+   "odahu-flow-mlflow-runner", "odahu/odahu-trainer", "Odahu-flow mlflow toolchain"
+   "odahu-flow-rest-packager", "odahu/odahu-packager", "Odahu-flow REST packager"
```
Comment (vlad-tokarev) [inline source/gen_distros.rst]: DO we need add airflow package here?

**Review by vlad-tokarev — APPROVED** [review]

File: source/dev_testing.md (hunk 4 above)
Comment (mcdoker18) [reply source/dev_testing.md:24]: We'll leave it at that for now.

File: source/comp_model_feedback.rst (hunk 3 above)
Comment (mcdoker18) [reply source/comp_model_feedback.rst]: Replaced

File: source/gs_feedback_loop.rst (hunk 6 above)
Comment (mcdoker18) [reply source/gs_feedback_loop.rst]: Replaced

File: source/gen_distros.rst (hunk 7 above)
Comment (mcdoker18) [reply source/gen_distros.rst]: Fixed

**Pull request merged**